docs/hub/repositories.md

```diff
@@ -0,0 +1,210 @@
+git lfs install
+```
+
+### Adding files to a repository
+
+Now's the time 🔥 You can add any files you want to the repository!
+
+You can do this with the usual Git workflow:
+
+```bash
+# Create any files you like! Then...
+git add .
+git commit -m "First model version"  # You can choose any descriptive message
+git push
+```
+
+And we're done! You can check your repository on Hugging Face with all the recently added files. For example, in the screenshot below the user added a number of files. Note that one of the files in this example has a size of `413.3 MB`, so the repo would use Git LFS to track it.
+
+![/docs/assets/hub/repo_with_files.png](/docs/assets/hub/repo_with_files.png)
+
+Every time you go through the `add`-`commit`-`push` cycle, the repo will keep track of every change you've made to your files. The UI allows you to explore the model files and commits and to see the difference (also known as *diff*) introduced by each commit:
+
+![/docs/assets/hub/explore_history.gif](/docs/assets/hub/explore_history.gif)
+
+
+### Renaming or transferring a repo
+
+
+If you own a repository, you will be able to visit the **Settings** tab to manage the name and ownership. Note that there are certain limitations in terms of use cases.
+
+Moving can be used in these use cases ✅ 
+- Renaming a repository within same user.
+- Renaming a repository within same organization. The user must be part of the organization and have "write" or "admin" rights in the organization.
+- Transferring repository from user to an organization. The user must be part of the organization and have "write" or "admin" rights in the organization.
+- Transferring a repository from an organization to yourself. You must be part of the organization, and have "admin" rights in the organization.
+- Transferring a repository from a source organization to another target organization. The user must have "admin" rights in the source organization **and** either "write" or "admin" rights in the target organization. 
+
+Moving does not work for ❌
+- Transferring a repository from an organization to another user who is not yourself.
+- Transferring a repository from a source organization to another target organization if the user does not have both "admin" rights in the source organization **and** either "write" or "admin" rights in the target organization.
+- Transferring a repository from user A to user B.
+
+If these are use cases you need help with, please send us an email at **website at huggingface.co**.
+
+
+### Dealing with large files
+
+Do you have files larger than 10MB? Those files should be tracked with `git-lfs`. When you use Hugging Face to create a repository, we automatically provide a list of common file extensions for these sfiles in `.gitattributes`, which `git-lfs` uses to efficiently track changes to your large files. However, you might need to add new extensions if your file types are not already handled. You can do so with `git lfs track "*.your_extension"`.
+
```

Comment:
I will remove this based on the comment from above (moving this to the uploading files section)